scripts/sort_includes.py

```diff
@@ -204,6 +203,12 @@ def print_sorted_includes(includes: list[str]):
             print(f"#include <{header}>")
         print()
 
+    if moc_header:
+        # moc include should come last -- this may rely on other includes
+        # to resolve forward declared classes
```

Comment:
Do the `moc_` includes really need to be last?
Isn't it enough that they are _after_ the file's header include?

Reply:
They definitely need to be last. Otherwise compilation breaks when something exposed as a qproperty has only been forward declared in the header.